### DISCUSSION TITLE
Windows+X menu skinned on Windows 10

### DISCUSSION THREAD

**Discussion by Jevil7452** [valinet/ExplorerPatcher#929] [General]
All other context menus get un-skinned, but not Windows+X menu.
I was able to reproduce the problem both on my main installation, and on a fresh VM.
The problem, as seen on my PC and a VM:
![pic1-problemvm](https://user-images.githubusercontent.com/98893064/155755896-3647021e-4eaa-4b4b-b9e2-2e38239e3e89.png)
![pic2-realpc](https://user-images.githubusercontent.com/98893064/155755898-896415df-0159-4354-919f-5fc335c5cc58.png)

Information:
Windows build number: 17763.2628 (my PC), 17763.316 (VM)
ExplorerPatcher: 22000.493.42.18

My settings:
![pic3-settings1](https://user-images.githubusercontent.com/98893064/155755977-858e9362-637a-401c-8a75-d709ff04f8aa.png)
![pic4-settings2](https://user-images.githubusercontent.com/98893064/155755985-819d1430-39e5-4f6e-9d83-e450b0fb9bd1.png)

**Comment by valinet** [comment]
Why is there a need to double post? Please delete the other post.

I will look into it, it’s because I don’t do anything regarding it on Windows 10 since originally EP doesn’t expect it to work, as on Windows 11.

**Comment by valinet** [comment]
Fixed in latest commit.
